Tasks/AzureFunctionAppV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -178,5 +179,10 @@
   "loc.messages.JarPathNotPresent": "Java jar path is not present",
   "loc.messages.FailedToUpdateApplicationInsightsResource": "Failed to update Application Insights '%s' Resource. Error: %s",
   "loc.messages.InvalidDockerImageName": "Invalid Docker hub image name provided.",
-  "loc.messages.MsBuildPackageNotSupported": "Deployment of msBuild generated package is not supported. Change package format or use Azure App Service Deploy task."
+  "loc.messages.MsBuildPackageNotSupported": "Deployment of msBuild generated package is not supported. Change package format or use Azure App Service Deploy task.",
+  "loc.messages.FailedToGetStorageAccountDetails": "Unable to find the storage account associated with the function app.",
+  "loc.messages.FailedToSyncTriggers": "Failed to sync triggers '%s' application settings. Error: %s",
```

Comment:
What does this message mean? "Application settings" seems to be disconnected in the sentence.

Reply:
@20shivangi , Can you please rephrase the sentence?
`Failed to sync triggers for web app '%s'. Error: %s`

Reply:
It is done.